charts/generic-prometheus-alerts/templates/rules.yaml

```diff
@@ -0,0 +1,31 @@
+{{- $targetApplication := required "A value for targetApplication must be set" .Values.targetApplication }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ $targetApplication }}-rules
+  labels:
+    {{- include "generic-prometheus-alerts.labels" . | nindent 4 }}
+spec:
+  groups:
+    - name: {{ $targetApplication }}-rules
+      rules:
+        - record: summer_time_offset
```

Comment:
might be a good idea to namespace these record rules e.g. 
`- record: {{ $targetApplication }}:summer_time_offset`

I'm not sure how prometheus would handle duplicate recording rules with the same name.  Same for the others below.

https://prometheus.io/docs/practices/rules/#naming-and-aggregation

Reply:
that's the bit I was asking about when I asked about global (public) rules. Yes, good idea